docs/docs/unexpected-input.mdx

```diff
@@ -72,68 +73,74 @@ def name(self):
     return "action_greet"
 
 def run(self, dispatcher, tracker, domain):
-    dispatcher.utter_template("utter_greet", tracker)
+    dispatcher.utter_message(template="utter_greet")
     return [UserUtteranceReverted()]
 ```
 
-To test the modified intents, we need to re-start our action server:
+This action will return the `utter_greet` response to the user, before removing the interaction from the dialogue history.
 
-```bash
-rasa run actions
-```
+#### 3. Creating a rule
 
-Then we can retrain the model, and try out our additions:
+If you've defined an action that includes `UserUtteranceReverted`, you only
+need to add a single rule for it, since it won't affect the dialogue history:
 
-```bash
-rasa train
-rasa shell
+```rasa-yaml
+rules:
+- rule: greet user
+  steps:
+  - intent: greet
+  - action: action_greet_user
 ```
 
-FAQs are another kind of generic interjections that should always get the same response.
-For example, a user might ask a related FAQ in the middle of filling a form:
-
+You'll also need to add any responses your action runs to your domain:
 
-
-<img alt="Generic Interjections" src={useBaseUrl("/img/generic_interjection.png")} width="240" />
-
-To handle FAQs defined with retrieval actions, you can add a simple story that will be handled by the MemoizationPolicy:
-
-```md
-## just sales, continue
-* contact_sales
-    - sales_form
-    - active_loop{"name": "sales_form"}
-* faq
-    - respond_faq
-    - sales_form
-    - active_loop{"name": null}
+```rasa-yaml
+responses:
+  utter_greet: Hi there!
 ```
 
-This will break out of the form and deal with the users FAQ question, and then return back to the original task.
-For example:
 
+### Generic Interjections
```

Comment:
Hmm, do we even need a generic interjections section? the rulepolicy should handle those automatically. I think it's only contextual interjections that we need to talk about here